tests/test_pseudocone.py

```diff
@@ -58,9 +56,49 @@ def test_list_test_data_users(mock_db_data, pseudocone_server):
     assert response.items[0].interactions[0].action == "ended"
 
 
+@patch("app.pseudocone.DatabaseClient.load_data", return_value=SINGLE_USER_SINGLE_ITEM_SINGLE_INTERACTION)
+def test_list_test_data_users_between_dates(mock_db_data, pseudocone_server):
+    # Should bring back test for data for a single user between the start and end date
+    users = [pseudocone_pb2.UserParam(id="1"), pseudocone_pb2.UserParam(id="2")]
+    request = pseudocone_pb2.ListTestDataUsersBetweenDatesRequest(limit=1,
+                                                                  users=users,
+                                                                  start_interaction_time="2018-03-01T17:53:32.000Z",
+                                                                  end_interaction_time="2018-03-01T19:53:32.000Z")
+
+    response = pseudocone_server["stub"].ListTestDataUsersBetweenDates(request=request)
+    assert len(response.items) == 1
+    assert response.items[0].user.id == "1"
+    assert len(response.items[0].interactions) == 1
+    assert response.items[0].interactions[0].pid == "b09s3kq5"
+    assert response.items[0].interactions[0].uri == "programmes:bbc.co.uk,2018/FIXME/b09s3kq5"
+    assert response.items[0].interactions[0].activity_time == "2018-03-01T18:53:32.000Z"
+    assert response.items[0].interactions[0].completion == "PT11M"
+    assert response.items[0].interactions[0].activity_type == "PLAYS"
+    assert response.items[0].interactions[0].action == "ended"
+
+    # Should return no items when activity time is before the start and end interaction time
```

Comment:
this part could be in its own test `test_list_test_data_users_between_dates_returns_no_items()`